src/vorta/utils.py

```diff
@@ -198,18 +201,19 @@ def get_private_keys() -> List[str]:
             if key.endswith('.pub') or key.startswith('known_hosts') or key == 'config':
                 continue
             try:
-                if is_ssh_file(key_file):
+                if is_ssh_private_key_file(key_file):
                     available_private_keys.append(key)
-            except (PermissionError, UnicodeDecodeError):
+            except PermissionError:
                 # Handling PermissionError separately from OSError because it can be safely ignored
-                # (If the user doesn't have permission to read the file, it's not an unexpected error)
-                # If UnicodeDecodeError is raised, it's because the file is not a valid SSH key
+                # (if the user doesn't have permission to read the file, it's not an unexpected error).
                 logger.debug(f'Expected error parsing file in .ssh: {key} (You can safely ignore this)', exc_info=True)
```

Comment:
Why is this "expected"?

Reply:
Maybe issueing a warning for the file would be better.

Reply:
Hmm, just noticed that this hasn't changed, it was "debug" and "expected" before this PR already.

Reply:
Your point still applies I guess

Reply:
This log line used to be for any private key error. It's much more specific now and would only trigger, when the current user can't read the file. If we want to keep the old behavior of logging anything that's not a private key, the line needs to go up:

```
if is_ssh_private_key_file(key_file):
    available_private_keys.append(key)
else:
    logger.debug("Not a private SSH key file...")
```

Reply:
Since we already have the permission check here, we could also check for correct key permissions. This sometimes comes up in support requests. Specifically SSH won't use the key if perms are **too open**. Like 644 or higher. Could be worth including and logging too.

Reply:
Guess it should check for `== 0o600` and else log a user-visible warning (not debug).

Reply:
Or even better: `mode & 0o077 == 0`.

Permissions for the owner could be r or rw (or even rwx).

Reply:
```python
logger.debug(f'Expected error parsing file in .ssh: {key} (You can safely ignore this)', exc_info=True)
```
should I replace this line then? With this:
```python
logger.debug(f'Permission error while opening file in .ssh: {key}', exc_info=True)
```

Reply:
Yes.